libs/labelbox/src/labelbox/schema/media_type.py

```diff
@@ -20,6 +20,8 @@ class MediaType(Enum):
     Unknown = "UNKNOWN"
     Unsupported = "UNSUPPORTED"
     Video = "VIDEO"
+    Point_Cloud = "POINT_CLOUD"
```

Comment:
what is point_cloud :)?

Reply:
a point cloud is a 3d scene with only points, no edges.